src/features/CodeLinkFeature.ts

```diff
@@ -24,10 +28,42 @@ const toggleCodeLinkActivationCommandName =
 	"hediet.vscode-drawio.toggleCodeLinkActivation";
 const linkCodeWithSelectedNodeCommandName =
 	"hediet.vscode-drawio.linkCodeWithSelectedNode";
-
+const linkSymbolWithSelectedNodeCommandName =
+	"hediet.vscode-drawio.linkSymbolWithSelectedNode";
+const linkWsSymbolWithSelectedNodeCommandName =
+	"hediet.vscode-drawio.linkWsSymbolWithSelectedNode";
 const linkFileWithSelectedNodeCommandName =
 	"hediet.vscode-drawio.linkFileWithSelectedNode";
 
+const symbolNameMap: Record<SymbolKind, string> = [
+	'symbol-file',
+	'symbol-module',
+	'symbol-namespace',
+	'symbol-package',
+	'symbol-class',
+	'symbol-method',
+	'symbol-property',
+	'symbol-field',
+	'symbol-constructor',
+	'symbol-enum',
+	'symbol-interface',
+	'symbol-function',
+	'symbol-variable',
+	'symbol-constant',
+	'symbol-string',
+	'symbol-number',
+	'symbol-boolean',
+	'symbol-array',
+	'symbol-object',
+	'symbol-key',
+	'symbol-null',
+	'symbol-enum-member',
+	'symbol-struct',
+	'symbol-event',
+	'symbol-operator',
+	'symbol-type-parameter'
+];
```

Comment:
Huh, I'm surprised this works!
I think it would be better like this though:
```
const symbolNameMap: Record<SymbolKind, string> = {
    SymbolKind.file: 'symbol-file',
    SymbolKind.module: 'symbol-module',
    ...
};
```

Otherwise I would need to review that the order of the symbols here matches with the numeric values of the SymbolKinds (which could change btw.).

Reply:
Yup, good idea. I assumed SymbolKinds enum would never change but that's a bad assumption.